WordPress/src/main/java/org/wordpress/android/ui/jetpack/BackupAvailableItemsProvider.kt

```diff
@@ -0,0 +1,61 @@
+package org.wordpress.android.ui.jetpack
+
+import androidx.annotation.StringRes
+import org.wordpress.android.R.string
+import org.wordpress.android.ui.jetpack.BackupAvailableItemsProvider.BackupAvailableItemType.CONTENTS
+import org.wordpress.android.ui.jetpack.BackupAvailableItemsProvider.BackupAvailableItemType.MEDIA_UPLOADS
+import org.wordpress.android.ui.jetpack.BackupAvailableItemsProvider.BackupAvailableItemType.ROOTS
+import org.wordpress.android.ui.jetpack.BackupAvailableItemsProvider.BackupAvailableItemType.SQLS
+import org.wordpress.android.ui.jetpack.BackupAvailableItemsProvider.BackupAvailableItemType.THEMES
+import org.wordpress.android.ui.jetpack.BackupAvailableItemsProvider.BackupAvailableItemType.PLUGINS
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * This class provides the available item choices for Jetpack Backup Download
+ */
+@Singleton
+class BackupAvailableItemsProvider @Inject constructor() {
```

Comment:
Idea (💡): What if we add a `Dummy` prefix on this provider's name. I recall @malinajirka also did that on one of his dummy providers. This can help us be consistent when using such providers. Wdyt?

Reply:
@ParaskP7  - It's not a dummy class - the data is not coming from the server, it is hardcoded as you see it. I added it to a provider because the potential exists for options to grow *AND* perhaps someday served by the API.

Reply:
Oh, my bad, for some reason I thought that this data will become dynamic, please ignore me! 😊 Many thanks for explicating that to me. 🙏